HISTORY.md

```diff
@@ -1,8 +1,8 @@
 # Version History
 
-## 1.1.9 / 2022-01-27
+## 1.1.10 / 2022-02-28
 
-- Updated for AVEVA Data Hub
+- Updated dependencies
```

Comment:
it looks like we lost the 1.1.9 entry when adding the 1.1.10 entry

Reply:
thank you!